README streamlining edits (#25)
AlexsLemonade/refinebio-examples: README.md

# refine.bio Examples Guide
This repository contains example workflows of how to use data downloaded from
[refine.bio](https://www.refine.bio).

## How to use:

If you are comfortable with GitHub, we recommend [cloning](https://help.github.com/articles/cloning-a-repository/)
this repository and following the examples.
You can also use the green button that says "Clone or download" to "Download 
ZIP" and then use these notebooks as you please on your own computer, if you 
prefer.

> Which is indicated toward the top of the page, in a gray button that says "Branch:".

Reviewer: Forgot to take this out when you took out the mention of master branch.

Author: Whoops.

# __refine.bio Examples Guide:__
This repository contains example workflows of how to use data downloaded from
[refine.bio](https://www.refine.bio).

## How to use:

If you are comfortable with GitHub, we recommend [cloning](https://help.github.com/articles/cloning-a-repository/)
this repository and following the examples.

> However, if you are less comfortable with GitHub, you can also use the green

Reviewer: ```suggestion
You can also use the green
```
And then end the sentence with "if you prefer."

button that says "Clone or download" to "Download ZIP" and then use these
notebooks as you please on your own computer.

> Make sure you are on the master branch.

Reviewer: If a person is less comfortable with Github, I think they are unlikely to be on a branch other than master and may find this confusing.

Which is indicated toward the top of the page, in a gray button that says "Branch:".

In each directory, there is a `README` that explains how to complete the
analyses with the R or GenePattern notebooks.
If you are following the instructions for completing the analyses in R, open the
R markdown (`Rmd`) and follow the instructions.
You will also see the example output files which are generally TSV or PNG files.
All the data used in these examples are deposited into the `data/`

> directory and are referenced as such in the R Notebooks.

Reviewer: I don't believe any of these are R Notebooks

https://github.com/AlexsLemonade/refinebio-examples/blob/cansav09/README_streamlining/clustering/clustering_example.Rmd#L5

Author: I changed all notebooks to be nb.html output now.

Author: Re-running the output files now and I'll push all those.

*For most of the example workflows, two general methods of completing the analyses are discussed:*  

### [R markdown](https://rmarkdown.rstudio.com/articles_intro.html)  
Workflows using R markdown have example analyses that could be applied to other
refine.bio formatted data.
The advantage of using these examples is that they were specifically for use
with refine.bio formatted data and associated metadata.

### [GenePattern notebooks](http://genepattern-notebook.org/example-notebooks/)
Workflows using GenePattern notebooks contain
[ready-made analyses](http://genepattern-notebook.org/example-notebooks/).
For users who are more comfortable with Python, or are not comfortable with
using scripts, the GenePattern notebooks are written in Python but can be
run using a graphics user interface (GUI).

> In order to use GenePattern notebooks, data from refine.bio needs to be converted

Reviewer: ```suggestion
For use with GenePattern notebooks, data from refine.bio needs to be converted
```

to GenePattern

> [formats](http://software.broadinstitute.org/cancer/software/genepattern/file-formats-guide).

Reviewer: Should we be explicit about the file format requirements here?

Author: The guide about formats that I have the link to I think covers all this well and there's no need for us to repeat it.

Reviewer: Honestly I meant to delete this comment before submitting 😆

The "gct" suffix will be added if you do not add it yourself.
Be sure to either have the script and input file in your current working
directory, or put type out the full directory path for the script and/or input
file. eg `/users/Bob/Desktop/create_gct_file.R`

#### CLS format file creation
CLS formatted files tell information regarding the groups or phenotype of the
different samples and are necessary for doing gene expression differential
analysis using GenePattern's notebooks. If you've already created a gct format
file from your data, you can create a a CLS format using GenePattern's online
[CLSFileCreator](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ClsFileCreator/4)

# Table of Contents  

1. [Clustering data](https://github.com/AlexsLemonade/refinebio-examples/blob/master/clustering)  

2. [Annotation using Ensembl IDs](https://github.com/AlexsLemonade/refinebio-examples/blob/master/ensembl-id-convert)  

3. [Differential Expression Analyses](https://github.com/AlexsLemonade/refinebio-examples/blob/master/differential-expression)  

4. [Ortholog Mapping](https://github.com/AlexsLemonade/refinebio-examples/blob/master/ortholog-mapping)  

> 5. [Batch Correction with ComBat](https://github.com/AlexsLemonade/refinebio-examples/blob/master/batch_correction)

Reviewer: This should get added in the same PR where the ComBat example gets added not before because it would introduce a broken link.
